test/rules/typedef/arrow-call-signature/test.ts.lint

```diff
@@ -18,7 +18,13 @@ var obj = {
 }
 
 setTimeout(() => {}, 1000);
+           ~~       [expected arrow-call-signature to have a typedef]
 setTimeout(function() {}, 1000);
 
+const foo = n => n+1
+          ~~~~       [expected arrow-call-signature to have a typedef]
+
 someFunc(n => n+1);
+        ~~~       [expected arrow-call-signature to have a typedef]
```

Comment:
same here, feels strange to include the `(` in the failure range

Reply:
It's made for (n) => n+1
.....................~~~       [expected arrow-call-signature to have a typedef]

The Node is only `n`.
But the code use `this.addFailure(location.pos - 1, location.end + 1, failure);`
I didn't touch this part